.travis/dispatch.sh

```diff
@@ -72,4 +72,10 @@ else
     if [[ "${SALT_NODE_ID}" == "servo-master1" ]]; then
         ./test.py sls.buildbot.master sls.homu sls.nginx
     fi
+
+    # Salt doesn't support timezone.system on OSX
+    # See https://github.com/saltstack/salt/issues/31345
+    if [[ "${TRAVIS_OS_NAME}" != "osx" ]]; then
```

Comment:
In the future the list of tests to run will be auto-generated from the Salt top file, so I'd prefer to check `${SALT_NODE_ID}` instead of `${TRAVIS_OS_NAME}`. You can use e.g. 
```
if [[ ! "${SALT_NODE_ID}" =~ servo-mac* ]]; then
   ...
fi
```